files/default/get_realm_status.groovy

```diff
@@ -0,0 +1,9 @@
+import org.sonatype.nexus.security.realm.RealmManager
+
+def realmManager = container.lookup(RealmManager.class.getName())
+
+if (realmManager) {
+    return realmManager.isRealmEnabled(args);
+} else {
+    return null;
```

Comment:
In which case do you think we'll enter this branch? I think you should fail if realmManager is not a valid instance. (Why are you more prudent on the get_realm_status than in the configure_realm script?)